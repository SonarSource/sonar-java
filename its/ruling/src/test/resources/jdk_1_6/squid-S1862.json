{
'jdk_1_6:java/io/ObjectInputStream.java':[
3483,
],
'jdk_1_6:java/text/MessageFormat.java':[
520,
535,
],
}
