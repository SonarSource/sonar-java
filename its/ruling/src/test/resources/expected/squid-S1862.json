{
'project:com/sun/org/apache/bcel/internal/generic/Type.java':[
264,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDTDScannerImpl.java':[
1468,
],
'project:com/sun/org/apache/xpath/internal/objects/XRTreeFrag.java':[
278,
],
'project:javax/swing/text/html/StyleSheet.java':[
2475,
],
}
